flight/targets/bl/common/main.c

```diff
@@ -449,9 +459,10 @@ int main(void)
 	} else if (PIOS_Boot_CheckRequest() == true) {
 		/* User has requested that we boot into firmware */
 		PIOS_IAP_ClearRequest();
-		PIOS_DELAY_WaitmS(1000);//needed so OS can detect BL USB disconnect
+		PIOS_DELAY_WaitmS(1000);//needed so OS can detect FW USB disconnect
```

Comment:
Yes, I think so.  We're immediately going around, so this avoids us missing the vid/pid coming back with the same usage.